src/main/java/commands/CommandLib.java

```diff
@@ -0,0 +1,171 @@
+package commands;
```

Comment:
<img width="1113" alt="Screenshot 2020-10-29 at 9 36 51 AM" src="https://user-images.githubusercontent.com/46109197/97514669-49f7d900-19ca-11eb-8909-9173ea9a8fa0.png">

Hi the functions in the sequence diagrams here are not linked to the description.  For example, get() of Command Library is not written in the description. Also, FoodItemAdd seems like a method and not a return value.